### PR TITLE
Added examples for remaining Jaeger Exporter options

### PR DESCRIPTION
**Pull request by kyle-lt**
Working on #1487.

## Changes

When reviewing the example [TestJaegerExporter.cs](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry.Exporter.Jaeger/JaegerExporterHelperExtensions.cs), I thought it'd be helpful to have some examples on how to configure the Jaeger Exporter options in addition to the existing host, port.

I added them in using default values (apart from the ProcessTags, which use arbitrary placeholders).

### CODE REVIEW

File: examples/Console/TestJaegerExporter.cs [hunk 1]
```diff
@@ -42,6 +44,24 @@ internal static object RunWithActivity(string host, int port)
                     {
                         o.AgentHost = host;
                         o.AgentPort = port;
+
+                        // Examples for the rest of the options, defaults unless otherwise specified
+                        o.MaxPayloadSizeInBytes = 4096;
+                        o.ProcessTags = new Dictionary<string, object>
```
Comment (CodeBlanch) [inline examples/Console/TestJaegerExporter.cs:50]: nit: Might be slightly more readable using index initializer syntax:
```csharp
                        o.ProcessTags = new Dictionary<string, object>
                        {
                            ["myKey1"] = "myVal1",
                            ["myKey2"] = "myVal2",
                        };
```

Comment (kyle-lt) [reply examples/Console/TestJaegerExporter.cs:50]: @CodeBlanch  - Cool, I made the change, thanks for the review!

File: examples/Console/TestJaegerExporter.cs [hunk 2]
```diff
@@ -42,6 +44,25 @@ internal static object RunWithActivity(string host, int port)
...
```
Comment (cijothomas) [inline examples/Console/TestJaegerExporter.cs:50]: Sorry for the delay in getting to this PR.
I am not sure of the need for `ProcessTags` option. We can simply use Resource attributes for this purpose, and Jaeger will automatically pick all tags from the Resource. So exposing another option here is not needed?

Comment (cijothomas) [reply examples/Console/TestJaegerExporter.cs:50]: Discussed with @CodeBlanch and agreed that this is not required any longer as Resource API is the official recommendation for additional tags.

I'll submit a PR to remove it. This PR looks good, once the ProcessTags section is removed. Thank you!

Comment (kyle-lt) [reply examples/Console/TestJaegerExporter.cs]: Sounds good @cijothomas , I just made the change to remove the process tags, appreciate it!

**Review by cijothomas — APPROVED** [review]

**Pull request merged**